recipes/nginx.rb

```diff
@@ -91,7 +90,7 @@
       mode '0644'
       variables(
         http_port: port,
-        server_name: site_opts['server_name']
+        server_name: site_opts['server_alias'].empty? ? site_name : site_opts['server_alias'].first
```

Comment:
I think it's a bit confusing to change from a user-provided setting to using the first alias for `ServerName`. Many, many use cases require setting this directly. I think trying to interpret it or prevent people from setting it directly is just going to frustrate people trying to use it. We're taking a simple thing and making it complicated, with no value add.